### PR TITLE
Reporter web app updates

### PR DESCRIPTION
**Pull request by tsteenbe**
See individual commits for details.

### CODE REVIEW

File: reporter-web-app/src/components/SummaryView.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Copyright (C) 2017-2021 HERE Europe B.V.
```
Comment (sschuberth) [inline reporter-web-app/src/components/SummaryView.js:2]: Commit message:

> so an user can quickly learn the amount

"a user", "learn about"

And you seem to have wrapped the commit message body text way before column 75.

File: reporter-web-app/src/components/SummaryView.js [hunk 2]
```diff
@@ -352,6 +352,10 @@ class SummaryView extends React.Component {
                                                 tab={(
                                                     <span>
                                                         Violations (
+                                                        {
+                                                            ruleViolations.length !== unresolvedRuleViolations
+                                                            && `${unresolvedRuleViolations}/`
```
Comment (sschuberth) [inline reporter-web-app/src/components/SummaryView.js:363]: Nit: Maybe spaces around `/`? Similar below.

File: reporter-web-app/src/App.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Copyright (C) 2017-2021 HERE Europe B.V.
```
Comment (sschuberth) [inline reporter-web-app/src/App.js:2]: Commit message: Could you maybe add a link to https://ant.design/components/icon/ so people get an idea what these icons look like beforehand?

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: reporter-web-app/src/components/SummaryView.js (hunk 2 above)
Comment (tsteenbe) [reply reporter-web-app/src/components/SummaryView.js:363]: I actually tried a version with spaces but it looked better without and o smaller screens the 2 spaces make the difference whether or not all tabs columns are shown.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**